force_wfmanager/central_pane/central_pane.py

```diff
@@ -23,16 +23,28 @@ class CentralPane(TraitsTaskPane):
     #: The plot view
     plot = Instance(Plot)
 
-    view = View(Tabbed(
+    #: Enable or disable the contained entities.
+    #: Used when the computation is running
+    enabled = Bool(True)
+
+    #: Button used to clear the model
+    clear_button = Button('Clear')
```

Comment:
Ok, this is apparently a good idea, but a very dangerous one. The problem is that one click can delete all data accumulated over hours of evaluation. They will still be in the file, but still...